packages/node_modules/@webex/react-component-confirmation-modal/src/index.js

```diff
@@ -30,6 +30,14 @@ function ConfirmationModal({
 }) {
   const modalClassNames = classNames('webex-dialogue-modal', styles.dialogueModal);
 
+  const modalRef = useRef(null);
+
+  useEffect(() => {
+    if (modalRef.current) {
+      modalRef.current.focus();
```

Comment:
We could rather use `<FocusLock/>` and `autoFocus` attrib there. That will let us remove all the `refs`

Reply:
@mkesavan13 since space-widgets is using node14 we can't use focus-lock right now. This is a way to solve this but I agree a better fix is to use focus-lock, but it is dependent on node version